### ISSUE TITLE
[REQUEST] Change ban loop evasion status

### ISSUE THREAD

**Issue by Enigma422** [openbullet/OpenBullet2#709]
Hi, I would like it if the ban loop evasion status can be customizable instead of being TOCHECK only.

**Comment by openbullet** [comment]
Hello, can you please provide a use case for this? Usually ban loop is for stuff you didn't account for, why would you like to mark that data anything but TO CHECK?

**Comment by openbullet** [comment]
Yeah sorry about it, soon I will finally have some time to work on issues again so I'm going through all the old ones.
